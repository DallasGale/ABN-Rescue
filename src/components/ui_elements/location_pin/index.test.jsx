/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import LocationPin from '.'

describe('<LocationPin />', () => {
  it('renders without crashing', () => {
    const component = render(<LocationPin />)
    expect(component).toMatchSnapshot()
  })
})
