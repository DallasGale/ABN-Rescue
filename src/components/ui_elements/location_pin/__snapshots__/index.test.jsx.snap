// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LocationPin /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="pin-circle-replacement exit
         styled__StyledPinCircle-sc-1axbwsm-1 dPPeUM"
        >
          <svg
            class="pin_circle size--large styled__StyledInnerCircle-sc-1axbwsm-4 cJWWTV"
            viewBox="0 0 590.72 590.72"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              class="styled__StyledSvgCircle-sc-1axbwsm-3 dMiWqf"
              cx="295.36"
              cy="295.36"
              r="295.36"
            />
          </svg>
        </div>
        <div
          class="styled__StyledPin-sc-1axbwsm-0 kyapFX"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="pin-circle-replacement exit
         styled__StyledPinCircle-sc-1axbwsm-1 dPPeUM"
      >
        <svg
          class="pin_circle size--large styled__StyledInnerCircle-sc-1axbwsm-4 cJWWTV"
          viewBox="0 0 590.72 590.72"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            class="styled__StyledSvgCircle-sc-1axbwsm-3 dMiWqf"
            cx="295.36"
            cy="295.36"
            r="295.36"
          />
        </svg>
      </div>
      <div
        class="styled__StyledPin-sc-1axbwsm-0 kyapFX"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
